app/src/main/java/nl/tudelft/b_b_w/view/MainActivity.java

```diff
@@ -25,12 +33,15 @@ protected void onCreate(Bundle savedInstanceState) {
         publicKey = "demokey";
 
         //test method
-        final int n = blockController.getLatestSeqNumber(ownerName);
-        Toast.makeText(this, "#" + n, Toast.LENGTH_LONG).show();
+        //final int n = blockController.getLatestSeqNumber(ownerName);
+        //Toast.makeText(this, "#" + n, Toast.LENGTH_LONG).show();
 
         // add genesis if we don't have any blocks
-        if (blockController.getLatestSeqNumber(ownerName) == 0)
-            addGenesis();
+        //if (blockController.getLatestSeqNumber(ownerName) == 0)
```

Comment:
I have removed the useless dead code. Thanks for notifying, it looks indeed a bit unclean.